includes/polling/os/f5.inc.php

```diff
@@ -1,20 +1,92 @@
 <?php
-$version = trim(snmp_get($device, '.1.3.6.1.4.1.3375.2.1.4.2.0', '-OQv'), '"');
```

Comment:
You've removed all the hardware info.  Why?

Reply:
I did not. I moved it to snmp_get_multi()

Reply:
Sorry, I missed it below.